src/scss/views/widgets/_image.scss

```diff
@@ -0,0 +1,5 @@
+.image-component {
```

Comment:
`component` is sorta implicit. let's call this something specific like
`circle-image` or something (i think that's what it is doing?)

Reply:
We could just call it image-wrapper, it's literally just a wrapper for images